res/database/Default/database_news.xml

```diff
@@ -3042,15 +3432,25 @@
 				<pl>Zwalczanie terroryzmu w przestrzeni kosmicznej</pl>
 			</title>
 			<description>
-				<de>US-Präsident Baush will zur Sicherheit den Terrorismus auch im All bekämpfen. Ein Spaceshuttle soll künftig 24h/Tag die int. Raumstation umkreisen. Kosten: 50Mrd USD.</de>
-				<en>Just to be on the safe side, US President Baush wants to fight terrorism also in space. A space shuttle is to orbit the int. space station 24 hours a day in the future. Cost: USD 50 billion.</en>
-				<pl>Na wszelki wypadek prezydent USA Baush chce walczyć z terroryzmem również w kosmosie. Prom kosmiczny ma w przyszłości orbitować wokół stacji kosmicznej 24 godziny na dobę. Koszt: 50 mld USD.</pl>
+				<de>US-Präsident ${presidentfirstname} ${presidentlastname} will zur Sicherheit den Terrorismus auch im All bekämpfen. Ein Spaceshuttle soll künftig 24h/Tag die Raumstation ${spacestationname} umkreisen. Kosten: 50Mrd USD.</de>
+				<en>Just to be on the safe side, US President ${presidentfirstname} ${presidentlastname} wants to fight terrorism also in space. A space shuttle is to orbit the space station ${spacestationname} 24 hours a day in the future. Cost: USD 50 billion.</en>
+				<pl>Na wszelki wypadek prezydent USA ${presidentfirstname} ${presidentlastname} chce walczyć z terroryzmem również w kosmosie. Prom kosmiczny ma w przyszłości orbitować wokół stacji kosmicznej ${spacestationname} 24 godziny na dobę. Koszt: 50 mld USD.</pl>
 			</description>
+			<variables>
+				<georgehw>${.person:"a7719914-657c-4269-947e-654e43ab1c3e":"firstname"}</georgehw>
+				<georgew>${.person:"1f75865b-39e1-4398-a6d9-ac35b6ac43f3":"firstname"}</georgew>
+				<bush>${.person:"1f75865b-39e1-4398-a6d9-ac35b6ac43f3":"lastname"}</bush>
+				<donald>${.person:"7cfb1ee1-0391-4532-8d85-7b70e24a6d9b":"firstname"}</donald>
+				<trump>${.person:"7cfb1ee1-0391-4532-8d85-7b70e24a6d9b":"lastname"}</trump>
+				<presidentfirstname>${.lt:${.worldtime:"year"}:1995:georgehw:"${.lt:${.worldtime:"year"}:2015:georgew:donald}"}</presidentfirstname>
+				<presidentlastname>${.lt:${.worldtime:"year"}:2015:bush:trump}</presidentlastname>
+				<spacestationname>${.lt:${.worldtime:"year"}:1998:"MIR":"ISS"}</spacestationname>
+			</variables>
 			<effects>
 				<effect trigger="happen" type="triggernews" news="7c2911a9-c9b4-40d1-b4f6-02fb0025358a" />
 			</effects>
-			<data genre="0" price="1.0" quality="58" />
-			<availability year_range_from="2001" year_range_to="2009" />
+			<data genre="0" price="1.0" quality="58" /> <!-- TODO: flags="2" but for each 'dumb' president; so rather just add delay or more entries to span several years -->
```

Comment:
This availability pattern could be bad for performance if used too often.
Option: the news disables itself and there are a couple of "non-visible" enabling news with happen time at the beginning of the next slot (possibly even with range)

Reply:
also there is a syntax error: `:2008}"` - the quotation mark must be removed

Reply:
thanks, can't test easily when it's inside a `script` attribute.